dashboard/config/routes.rb

```diff
@@ -158,6 +158,7 @@ def redirect_to_teacher_dashboard
   get '/stats/:user_id', to: 'reports#user_stats', as: 'user_stats'
   get '/stats/level/:level_id', to: 'reports#level_stats', as: 'level_stats'
   get '/popup/stats', to: 'reports#header_stats', as: 'header_stats'
+  get '/popup/progress', to: 'reports#user_progress', as: 'user_progress'
```

Comment:
I called this route /popup/progress because /popup/stats is also the header.
Probably a bad name - both of these APIs can move together because the /popup/stats one will also become an API.